wait-for.c

```diff
@@ -270,15 +268,18 @@ int main(int argc, const char **argv) {
 		goto exit;
 	}
 
+	dirpath_buffer = strdup(extrav[0]);
```

Comment:
`strdup` isn't available everywhere. Please make a stub of the function and use it instead.

Reply:
Still many systems are missing `strdup` :) I run into it _frequently_ when cross-compiling.